src/hooks/prerun/update-brew-check.ts

```diff
@@ -0,0 +1,23 @@
+import { Hook } from "@oclif/core";
+import { execSync } from "child_process";
+
+const hook: Hook<"prerun"> = async function (opts) {
+  const isInstalledWithBrew = () => {
+    try {
+      const cellar = execSync("brew --cellar", { encoding: "utf8" });
+      return opts.config.root.startsWith(cellar);
+    } catch {
+      return false;
+    }
+  };
+
+  if (opts.Command.id === "update") {
+    if (isInstalledWithBrew()) {
+      opts.context.error(
+        "installed with brew.\nUse `brew upgrade mw` to update to the newest version",
+      );
```

Comment:
This will trigger a full-blown error message. Changing to something less intimidating:


```suggestion
      opts.context.log(
        "It looks like the mw CLI was installed using Homebrew.\nPlease use `brew upgrade mw` to update to the newest version",
      );
      opts.context.exit(1);
```

WDYT?

Reply:
From the looks of it, I think `.warn` might be a good compromise.

Alternatively, if we use error, we should customize the rendering for that specific error, because the default renderer is more intended for unexpected exceptions (which is why it also suggests the GitHub link and gives you the stack trace), and not for anticipated errors like these.